pynestml/codegeneration/resources_python_standalone/point_neuron/directives/StateVariablesEnum.jinja2

```diff
@@ -0,0 +1,20 @@
+{#
+  Generates an Enum with state variables that are recordable when the neuron model uses vectors
+  @param neuron ASTNeuron: the neuron model
+-#}
+
+enum StateVecVars {
```

Comment:
Are vectors supported for the python-standalone target?

Reply:
Probably this would not be very difficult to implement as indeed a lot of the prerequisites are already there, but it would require adding at least some unit tests, if it's alright with you I'd like to save that for a separate PR.